pkg/operator/config.go

```diff
@@ -67,6 +97,24 @@ func ConfigFromEnvironment() *Config {
 		config.WatchNamespace = watchNamespace
 	}
 
+	if leaderElection, ok := os.LookupEnv("LEADER_ELECTION"); ok {
+		le, err := strconv.ParseBool(leaderElection)
+		if err != nil {
+			le = DefaultLeaderElection
+			glog.Errorf("Error parsing LEADER_ELECTION environment variable: %v", err)
```

Comment:
Is the intention here that this falls through and `le` is (possibly) `false` (depending on implementation)?

Reply:
If the environment variable is present, but fails to parse, we log the error and use the default value. That happens to be `true` at the moment.

Reply:
Ah, sorry. Missed the `le = DefaultLeaderElection`. Somehow!